pom.xml

```diff
@@ -159,7 +159,7 @@
     <httpcomponents.client.version>4.5.13</httpcomponents.client.version>
     <httpcomponents.core.version>4.4.13</httpcomponents.core.version>
     <ivy.version>2.4.0</ivy.version>
-    <jackson.version>2.12.0</jackson.version>
+    <jackson.version>2.10.0</jackson.version>
```

Comment:
@Aggarwal-Raghav  Rather than downgrading jackson, can we also look for option to upgrade arrow version itself to stabilize the tests.

Reply:
Have upgraded arrow to 0.10.0